src/data_processing/wiki_to_netflix.py

```diff
@@ -115,14 +114,17 @@ def process_data(test=False):
     missing_count = 0
     processed_data = []
 
-    netflix_data = read_netflix_txt(os.path.join(data_dir, 'movie_titles.txt'), test)
-    num_rows = len(netflix_data)
+    netflix_file = read_netflix_txt(os.path.join(data_dir, 'movie_titles.txt'), test)
 
     netflix_csv = os.path.join(out_dir, 'movie_titles.csv')
 
-    wiki_movie_ids_list, wiki_genres_list, wiki_directors_list = wiki_query(netflix_data, user_agent)
+    wiki_movie_ids_list, wiki_genres_list, wiki_directors_list = wiki_query(netflix_file, user_agent)
+
+    num_rows = len(wiki_movie_ids_list)
+
+    netflix_file = read_netflix_txt(os.path.join(data_dir, 'movie_titles.txt'), test)
```

Comment:
Looks like this is duplicated from above. I think you want to remove this line?

Reply:
Yup it has been removed